app/src/main/java/de/danoeh/antennapod/fragment/preferences/ImportExportPreferencesFragment.java

```diff
@@ -170,7 +170,7 @@ private void exportDatabase() {
             File backupDB = new File(sd, dateStampFilename(DATABASE_EXPORT_FILENAME));
             progressDialog.show();
             disposable = Completable.fromAction(() ->
-                        DatabaseExporter.exportToStream(new FileOutputStream(backupDB), getContext()))
+                    DatabaseExporter.exportToStream(new FileOutputStream(backupDB), getContext()))
```

Comment:
This change makes it harder to see what belongs to the lambda and what is outside the lambda. Please revert.

Reply:
It's strange. I didn't change this line
![image](https://user-images.githubusercontent.com/3222305/95020190-941bc200-068b-11eb-8f11-1cda1a3cc006.png)

Reply:
I don't know. Maybe you branched from an older version? Just change it so that GitHub does not display a change :)